ui/shared/components/form/AnvilFileSelector.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { HttpRequestHelper } from 'shared/utils/httpRequestHelper'
+import { Select } from 'shared/components/form/Inputs'
+import { FormSpy } from 'react-final-form'
+import DataLoader from '../DataLoader'
+
+class AnvilFileSelector extends React.PureComponent {
+
+  static propTypes = {
+    namespace: PropTypes.string.isRequired,
+    name: PropTypes.string.isRequired,
+  }
+
+  state = {
+    dataPathList: null,
+    loading: false,
+  }
+
+  load = () => {
+    const { namespace, name } = this.props
+    this.setState({ loading: true })
+    new HttpRequestHelper(`/api/create_project_from_workspace/${namespace}/${name}/get_vcf_list`,
+      (responseJson) => {
+        this.setState({ loading: false, dataPathList: responseJson.dataPathList || [] })
+      },
+      (e) => {
+        this.setState({ loading: false, dataPathList: [e.message] })
+      }).get()
+  }
+
+  render() {
+    const { dataPathList, loading } = this.state
+    const options = dataPathList?.map(file => ({ name: file, value: file })) || []
```

Comment:
We need to show the user an error if there are no valid files to choose from - right now they would just have a dropdown with no options to choose from which would be a kind of bad experience. Lets do the following:
```
<Message
    error
    header="No Joint Called VCF found in workspace"
    content="There are no joint called VCFs in the Files section of this workspace. VCFs must have a .vcf, .vcf.gz, or .vcf.bgz file extension. Please add a VCF to your workspace before proceeding with loading."
  />
```